test/evergreen.yml

```diff
@@ -2270,53 +2274,21 @@ tasks:
       - func: "get project"
       - func: "compile wiredtiger"
         vars:
+          HAVE_UNITTEST: -DHAVE_UNITTEST=1
           <<: *configure_flags_with_builtins
           CMAKE_TOOLCHAIN_FILE: -DCMAKE_TOOLCHAIN_FILE=../cmake/toolchains/mongodbtoolchain_v3_gcc.cmake
           CMAKE_BUILD_TYPE: -DCMAKE_BUILD_TYPE=Coverage
-      - func: "make check all"
       - func: "unit test"
         vars:
-          unit_test_args: -v 2 --long
```

Comment:
I'm curious how much coverage we could gain back if leaving the long option. The runtime of the coverage test is not a big concern to me (as long as it runs into timeout failure).

Reply:
I gave it a try earlier and it was less than 0.1%. 
Here are the patches for [with --long](https://spruce.mongodb.com/version/62e310a4a4cf470f9c71b588/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC), and [without --long](https://spruce.mongodb.com/version/62e5dbe22a60ed38397ca6b9/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC).

Reply:
Good to know the difference is neglegible.